.github/workflows/soak.yml

```diff
@@ -44,6 +44,10 @@ jobs:
       soak-cpus: ${{ steps.system.outputs.SOAK_CPUS }}
       soak-memory: ${{ steps.system.outputs.SOAK_MEMORY }}
     steps:
+      - uses: actions/checkout@v2.3.5
+        with:
+          path: comparison-vector
```

Comment:
So is the default here to checkout the PR's branch merged into the base? If so, that might be worth a comment.

Reply:
Yeah, that's right, and agreed. I'll add a comment.